lrauv_system_tests/worlds/flat_seabed.sdf

```diff
@@ -23,6 +23,14 @@
       filename="ignition-gazebo-physics-system"
       name="gz::sim::systems::Physics">
     </plugin>
+    <plugin
```

Comment:
Knit: We don't need this for unit tests as the user does not need to visualize the scene.

Reply:
Agreed. At the same time, it's super handy to be able to just do `gz sim` on a test world when a test is failing. Maybe we can comment it out? WDYT?